Library/Homebrew/utils/analytics.sh

```diff
@@ -14,17 +22,15 @@ setup-analytics() {
   if [[ "$(git config --file="$HOMEBREW_REPOSITORY/.git/config" --get homebrew.analyticsmessage)" != "true" ||
         "$(git config --file="$HOMEBREW_REPOSITORY/.git/config" --get homebrew.analyticsdisabled)" = "true" ]]
   then
```

Comment:
We should probably remove the `homebrew.analyticsuuid` setting from the git config when disabling analytics here, to preserve the old behavior of getting rid of the UUID.